src/pages/resume-page.md

```diff
@@ -0,0 +1,146 @@
+  - image: '../public/usedresources/nextjs.svg'
+    alt: 'Next.js'
+    name: Next.js
+  - image: '../public/usedresources/angular.svg'
+    alt: 'Angular'
+    name: Angular
+  - image: '../public/usedresources/vue.svg'
+    alt: 'Vue.js'
+    name: Vue.js
+  - image: '../public/usedresources/github-black.svg'
+    alt: 'Github'
+    name: Github
+  - image: '../public/usedresources/git.svg'
+    alt: 'Git'
+    name: Git
+  - image: '../public/usedresources/azure.svg'
+    alt: 'Azure Dev Ops'
+    name: Azure Dev Ops
+  - image: '../public/usedresources/nunjucks.svg'
+    alt: 'Nunjucks'
+    name: Nunjucks
+  - image: '../public/usedresources/handlebars.svg'
+    alt: 'Handlebars'
+    name: Handlebars
+  - image: '../public/usedresources/jest.svg'
+    alt: 'Jest'
+    name: Jest
+  - image: '../public/usedresources/cypress.svg'
+    alt: 'Cypress'
+    name: Cypress
+  - image: '../public/usedresources/figma.svg'
+    alt: 'Figma'
+    name: Figma
+
+currentExperienceHighLevels:
+  - classes: cmp-resume-page__list-item--heading
+    content: |
+      January 2023 &hyphen; Present
+  - classes: cmp-resume-page__list-item--title util-bold
+    content: Developer
+  - classes: util-margin-none
+    content: Sparkbox
+  - classes: util-margin-none
+    content: |
+      Bellbrook, <abbr title="Ohio">OH</abbr>
+currentExperienceDetails:
+  - content: 'Learning cohorts, Accessibility Guild, Foundry articles (Contrast), Lightning Talk presentation (React Design Patterns), internal demos (Figma recreations), internal tooling, client demos'
+  - content: 'Projects: NCSBN Global Regulatory Atlas, Propulsion Design System Builder, NCSNB.org Redesign, Crown Design System Documentation, sparkbox.com (migrate static to dynamic pages, testing, Github actions, documentation)'
+  - content: 'Technologies: HTML, CSS, SCSS, React, JavaScript, Docker, Git, GitHub, Jest, Cypress, Backstop, Vitest, Angular, Vue.js, Jira, Azure DevOps, Handlebars, Bootstrap, Netlify, Figma, Nunjucks, TypeScript, Next.js'
+
+secondaryExperienceHighLevels: 
+  - classes: cmp-resume-page__list-item--heading
+    content: |
+      July 2022 &hyphen; December 2022
+  - classes: cmp-resume-page__list-item--title util-bold
+    content: Apprentice Full-Stack Developer
+  - classes: util-margin-none
+    content: Sparkbox
+  - classes: util-margin-none
+    content: |
+      Bellbrook, <abbr title="Ohio">OH</abbr>
+secondaryExperienceDetails:
+  - content: Weekly 1:1&apos;s with mentor and technical directors, pairing sessions with Production team, monthly reviews, curriculum discussions
+  - content: 'Philosophies: functional programming, object composition, test-driven development, responsive design, WCAG 2.1, <abbr title="Inverted Triangle CSS">ITCSS</abbr>'
+  - content: 'Projects: Sparkhero, Accessible Components, Pokemon Clash!'
+  - content: 'Technologies: HTML, CSS, Git, CLI, Jira, GitHub, SCSS, React, JavaScript, MySQL Workbench, Next.js, Jest, Docker, Cypress'
```

Comment:
Let's get rid of the technologies bit of each job. It's not necessary if the technologies are listed over in the "Tech" column.